src/constants/constants.js

```diff
@@ -38,6 +38,75 @@ export default {
     }
   },
 
+  git: {
```

Comment:
I really think this will get unmanageable. We should move these huge template strings to its own file. You could create a new file for these template strings.

Reply:
Talk to Jack on this one.

Reply:
I agree. We should move some of these long constants to their own files. Probably anything that creates a file should be on its own.